sites/cheerpx/src/content/docs/00-overview.mdx

```diff
@@ -23,22 +23,22 @@ import LinkButton from "@leaningtech/astro-theme/components/LinkButton.astro";
 
 CheerpX enables you to run existing 32-bit x86 native binaries and Linux distributions:
 
-- without modifications;
-- without recompilation;
-- without source-code;
-- fully client-side.
+- Without modifications
+- Without recompilation
+- Without source-code
+- Fully client-side
 
-CheerpX is perfect for web-based IDEs, programming tutorials, virtual machines, and interactive demos. It can also be used to provide a secure sandbox for untrusted code. CheerpX is used by [WebVM](https://webvm.io/), [PythonFiddle](https://pythonfiddle.leaningtech.com/), and has been fully tested in production as part of [CheerpX for Flash](/cheerpx-for-flash), an enterprise solution to run legacy Adobe Flash content in the browser.
+CheerpX is perfect for web-based IDEs, programming tutorials, virtual machines, and interactive demos. It can also be used to provide a secure sandbox for untrusted code. CheerpX is very robust and deployed as part of [WebVM](https://webvm.io/) and [PythonFiddle](https://pythonfiddle.leaningtech.com/). It is also used in production environments as part of [CheerpX for Flash](/cheerpx-for-flash), an enterprise solution to run legacy Adobe Flash content in the browser.
```

Comment:
```suggestion
CheerpX is perfect for web-based IDEs, programming tutorials, virtual machines, and interactive demos. It can also be used to provide a secure sandbox for untrusted code. CheerpX is very robust and deployed as part of [WebVM](https://webvm.io/) and [PythonFiddle](https://pythonfiddle.leaningtech.com/). It is also used in production environments as part of [CheerpX for Flash](https://labs.leaningtech.com/docs/cheerpx-for-flash), an enterprise solution to run legacy Adobe Flash content in the browser.
```